File: coffee-system/src/main/java/com/jesperancinha/coffee/system/concurrency/StartupCallableImpl.java
package com.jesperancinha.coffee.system.concurrency;

/**
 * Created by joao on 8-5-16.
 */
public class StartupCallableImpl  extends  QueueCallableAbstract implements  StartupCallable {
    @Override
    public Boolean call() throws Exception {
        return true;
    }
}

File: coffee-system/src/main/kotlin/org/jesperancinha/coffee/system/concurrency/StartupCallableImpl.kt
package org.jesperancinha.coffee.system.concurrency

import org.jesperancinha.coffee.system.api.concurrency.QueueCallable
import java.util.concurrent.Callable

/**
 * Created by joao on 8-5-16.
 */
class StartupCallableImpl : QueueCallableAbstract(), QueueCallable {
    @Throws(Exception::class)
    override fun call() = true
}